metaflow/plugins/kfp/tests/run_integration_tests.py

```diff
@@ -159,6 +171,7 @@ def test_error_and_opsgenie_alert(pytestconfig) -> None:
         f"--wait-for-completion --workflow-timeout 1800 "
         f"--experiment metaflow_test --tag test_t1 "
         f"--error_flow_id={error_flow_id} "
+        f"--notify "
```

Comment:
if we have notify for all flows, then we don't test without notify too....

Reply:
Happy to remove `--notify` in the general tests however if you want them tested without `--notify`. We already have the `s3_sensor` flows tested with `--notify`. Alternatively, we could have every test tested with and without notify (doubling the number of tests), but I don't think this is time or cost effective.

Reply:
At least one Flow without `--notify` would be necessary to cover the test matrix